src/middlewared/middlewared/plugins/zfs.py

```diff
@@ -99,6 +99,13 @@ def query_imported_fast(self):
         except libzfs.ZFSException as e:
             raise CallError(f'Failed listing imported pools with error: {e}')
 
+    def is_upgraded(self, pool_name):
+        with libzfs.ZFS() as zfs:
+            for pool in filter(lambda x: x.name == pool_name, zfs.pools):
+                return all((i.state.value in (1, 2) for i in pool.features))
```

Comment:
Why? I meant `i.state in (libzfs.FeatureState.ENABLED, libzfs.FeatureState.ACTIVE)`, will it work?